lib/ecto/adapters/postgres/connection.ex

```diff
@@ -672,6 +668,7 @@ if Code.ensure_loaded?(Postgrex.Connection) do
 
     defp assemble(list) do
       list
+      |> Enum.reject(& &1 == "")
```

Comment:
Please don't add this. Just make "" and empty list and it will be flattened out. :)